### ISSUE TITLE
APIServer CommandLine Flags on klog(glog) seems applied as default(initial) value

### ISSUE THREAD

**Issue by dormael** [kubernetes-incubator/apiserver-builder-alpha#387]
I found -v flag is not affecting when starting api server.

Following modification fixed the problem.
But not sure if this is proper way of modification.
Cause I'm new to golang and kubernetes.

```
diff --git a/pkg/cmd/server/start.go b/pkg/cmd/server/start.go
index 2ca42994d..45f76017b 100644
--- a/pkg/cmd/server/start.go
+++ b/pkg/cmd/server/start.go
@@ -190,15 +190,7 @@ func NewCommandStartServer(etcdPath string, out, errOut io.Writer, builders []*b
 
 	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
 	klog.InitFlags(klogFlags)
-	flags.AddGoFlagSet(klogFlags)
-
-	// Sync the glog and klog flags.
-	klogFlags.VisitAll(func(f *flag.Flag) {
-		goFlag := flag.CommandLine.Lookup(f.Name)
-		if goFlag != nil {
-			goFlag.Value.Set(f.Value.String())
-		}
-	})
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 
 	return cmd, o
```

**Comment by yue9944882** [comment]
thank you for reporting this, just confirmed in my local environment. and i just sent a fix for this issue #388

**Comment by yue9944882** [comment]
for now, we're vendoring 1.12 kubernetes, it's an awkward version in the middle of migrating from glog to klog.
